src/Components/NavbarComponent/DrawerComponents/ProfileDrawerComponent/ProfileDrawerComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import {
+  Drawer,
+  List,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  type Theme,
+  useTheme,
+} from "@mui/material";
+import Brightness4Icon from "@mui/icons-material/Brightness4";
+import Brightness7Icon from "@mui/icons-material/Brightness7";
+import {
+  COLORMODECONTEXT,
+  type IThemeContext,
+} from "../../../../Context/ThemeContext";
```

Comment:
We might need to find a better way to do those imports ...

Reply:
In webpack, you can set imports aliases : https://webpack.js.org/configuration/resolve/.
For example, `src` could be aliased to `@`, this import could then be translated to `@/Context/ThemeContext`.

According to [this post](https://stackoverflow.com/a/65790801) this is also possible with CRA.

I believe vite has this exact alias by default.